packages/next/src/server/base-server.ts

```diff
@@ -3054,7 +3054,11 @@ export default abstract class Server<
       }
     )
 
-    if (isPrefetchRSCRequest && typeof segmentPrefetchHeader === 'string') {
+    if (
+      isRoutePPREnabled &&
```

Comment:
Let's use `clientSegmentCache` flag instead. We do per-segment requests for static pages too.

Reply:
Discussed offline, but `clientSegmentCache` wasn't plumbed to the server yet in 15.1 and since it wasn't ready in this release we can safely just disable on PPR